styles/desktop.css

```diff
@@ -0,0 +1,347 @@
+    background-size: contain;
+    background-size: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    align-items: center;
+  }
+
+  .intro {
+    width: 100%;
+    height: 100%;
+    padding: 5rem 1rem;
+  }
+
+  .intro h2 {
+    font-family: "Lato", 'Times New Roman', Times, serif;
+    font-size: 2rem;
+    width: 60%;
+    font-weight: 700;
+    color: var(--soft-orange);
+  }
+
+  .intro h2::before {
+    content: '"';
+  }
+
+  .intro h2::after {
+    content: '"';
+  }
+
+  .intro h1 span {
+    font-size: 3rem;
+  }
+
+  .intro h1 {
+    width: 60%;
+    font-family: Cocogoose, 'Lato', 'Times New Roman', Times, serif;
+    font-size: 4rem;
+  }
+
+  .intro p {
+    width: 60%;
+    margin-top: 1rem;
+    background-color: #f0e8e8e7;
+    border: solid 5px var(--hard-white);
+    padding: 1rem;
+  }
+
+  .intro .dates {
+    font-size: 2.5rem;
+    margin-top: 1rem;
+  }
+
+  .intro h4 {
+    width: 60%;
+  }
+
+  .the {
+    display: none;
+  }
+
+  .f-1 {
+    display: block;
+    font-size: 4rem;
+    color: var(--soft-orange);
+    font-weight: 700;
+    font-family: "Cocogoose", 'Lato', 'Times New Roman', Times, serif;
+  }
+
+  .events {
+    display: grid;
+    width: 95%;
+    height: 15.5rem;
+    grid-template-columns: repeat(5, 20%);
+    margin-top: 2rem;
+    justify-content: center;
+    gap: 0;
+  }
+
+  .event-link {
+    display: block;
+    text-decoration: none;
+    color: var(--hard-white);
+    font-family: 'Lato', 'Times New Roman', Times, serif;
+    font-weight: 400;
+    margin-top: 4rem;
+    border-bottom: solid 1px var(--hard-white);
+  }
+
+  main {
+    margin-top: -1rem;
+  }
+
+  .activity {
+    width: 99%;
+    padding: 1rem 0.25rem;
+    list-style: none;
+    background-color: rgba(80, 82, 88, 0.7);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    color: var(--white);
+    font-family: "Lato", 'Times New Roman', Times, serif;
+    text-align: center;
+  }
+
+  .activity:hover {
+    border: solid 2px var(--hard-white);
+  }
+
+  main button {
+    display: none;
+  }
+
+  .list-title {
+    color: var(--f1-red);
+  }
+
+  .list-desc {
+    font-weight: 400;
+  }
+
+  .list-img img {
+    width: 3.75rem;
+    height: 3.75rem;
+  }
+
+  .extra {
+    display: block;
+  }
+
+  #card-container {
+    display: grid;
+    grid-template-columns: repeat(2, 50%);
+    width: 100%;
+    padding: 1rem 5rem;
+    justify-items: center;
+    align-items: center;
+  }
+
+  .racer {
+    width: 80%;
+  }
+
+  .partners {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: none;
+    background-color: var(--capstone-grey);
+    justify-items: center;
+    align-items: center;
+    padding: 1rem;
+    gap: 0;
+    margin-top: 2rem;
+    width: 90%;
+  }
+
+  .partners img {
+    width: 80%;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
```

Comment:
Your Implementation|Provided design
-----|----
<img width="367" alt="Screenshot 2023-08-02 at 12 15 27 PM" src="https://github.com/Chimoneg27/Capstone-One/assets/44798044/56b4e265-a0fa-4116-8cfd-e4d921b19958">|<img width="483" alt="Screenshot 2023-08-02 at 12 26 58 PM" src="https://github.com/Chimoneg27/Capstone-One/assets/44798044/7b136807-c7dc-4f05-be93-241d54965657">


- [ ] Kindly align your partner images properly to match the provided design currently there is too much gap between the logo images and also there is too little padding from the left. I would recommend using flex-box instead of grid for aligning the partners section as this would be more suitable in this case. Consider the following suggestion.

```suggestion
  .partners {
    display: flex;
    flex-wrap: wrap;
    background-color: var(--capstone-grey);
    justify-content: center;
    padding: 1rem;
    gap: 1rem;
    margin-top: 2rem;
  }

  .partners img {
    width: 70px;
  }

```

Result:
---
<img width="360" alt="Screenshot 2023-08-02 at 12 25 12 PM" src="https://github.com/Chimoneg27/Capstone-One/assets/44798044/9baf8a1b-a540-4b2e-90fd-43277ced85b6">